components/Breadcrumb/src/stories/breadcrumb.bem.stories.mdx

```diff
@@ -0,0 +1,896 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+
+import { Breadcrumb } from "./../index";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+
+<Meta
+  title="CSS Components/Navigation/Breadcrumb"
+  component={Breadcrumb}
+  parameters={{
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    docs: { source: { type: "dynamic" } },
+    status: {
+      type: "WORK IN PROGRESS",
+    },
+  }}
+/>
+
+<Description markdown={readme} />
+
+## Design Tokens
+
+<DesignTokenDocBlock categoryName="Breadcrumb" viewType="table" />
+
+## Stories
+
+### Default
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Default">
+    <nav class="denhaag-breadcrumb">
```

Comment:
[WAI ARIA](https://www.w3.org/TR/wai-aria-practices/examples/breadcrumb/index.html) practices suggest an `aria-label="Breadcrumb"` on the `<nav>` element.

Reply:
Applies to the other stories as well

Reply:
Will do